src/core/Akka.TestKit.Tests/Bugfix7145Spec.cs

```diff
@@ -0,0 +1,55 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="Bugfix7145Spec.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2024 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2024 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Akka.Actor;
+using FluentAssertions;
+using Xunit;
+
+namespace Akka.TestKit.Tests;
+
+public class Bugfix7145Spec : AkkaSpec
```

Comment:
This spec was able to reproduce the deadlock reported in #7145 on `dev` - changes so far allows these specs to pass.